sdk/python/kfp/onprem.py

```diff
@@ -1,3 +1,5 @@
+from typing import Dict, Text
```

Comment:
Since we no longer support Python 2 maybe we can drop `Text`?

Reply:
Pushed fix.